src/System.Text.Json/pkg/Microsoft.Bcl.System.Text.Json.Sources.pkgproj

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <UsingTask TaskName="GenerateSourceWithPublicTypesAsInternal" AssemblyFile="$(PackagingTaskDir)Microsoft.DotNet.Build.Tasks.Packaging.dll" />
+
+  <PropertyGroup>
+    <!-- We need to make sure that all of the surface area on the sources gets rewritten as internal. -->
+    <GenerateInternalTypesSource>true</GenerateInternalTypesSource>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <_AllJsonSources Include="$(MSBuildThisFileDirectory)../src/**/*.cs" />
+    <SourcePackageFiles Include="@(_AllJsonSources)">
+      <PackagePath>contentFiles/cs/netstandard1.1/%(RecursiveDir)%(FileName)%(Extension)</PackagePath>
```

Comment:
Based on https://github.com/dotnet/corefx/issues/34577, this should be netstandard2.0 (since that is what we plan to support).

Reply:
sounds good, I'll go ahead and change that.